aspnetcore/release-notes/aspnetcore-9/includes/hybrid-cache.md

```diff
@@ -6,6 +6,9 @@ ms.date: 05/21/2024
 ---
 ### New `HybridCache` library
 
+(Note: `HybridCache` will be fully released in the dotnet/extensions 9.1 release shortly *after* .NET 9.0; in the 9.0 release, it will be available and usable, but
+in an unsupported preview state, and will have the `[Experimental]` marker.)
```

Comment:
```suggestion
> [!IMPORTANT]
> `HybridCache` is currently still in preview but will be fully released shortly *after* .NET 9.0 with .NET Extensions 9.1.)
```

Reply:
@mgravell @adityamandaleeka How confident are we that this will ship with .NET Extensions 9.1? Should we avoid making any specific release commitment in case plans need to change?

Reply:
+1, I would probably avoid mentioning specific releases and instead simply say something along the lines of: `available in a future minor release of dotnet/extensions` or something like that.

Reply:
Whatever we put in the **What's New** we should put also in the feature documentation page.  I did that just now, so we'll need to change the note in both places if we change the wording to eliminate the promise of 9.1.